### ISSUE TITLE
Create an organization or move under another organization

### ISSUE THREAD

**Issue by Arcterus** [DaGenix/rust-crypto#387]
I think it would be better for this repository to be under the control of an organization so that pull requests and issues can be dealt with even while @DaGenix is busy.  Currently there are 21 pull requests that need to be sorted through.

Thoughts?

**Comment by burdges** [comment]
I'm inclined to migrate to [ring](https://github.com/briansmith/ring) actually.

**Comment by burdges** [comment]
Agreed.  Alternatives like [ring](https://github.com/briansmith/ring) or sodiumoxide expose way too little.

**Comment by burdges** [comment]
Can [rust-lang-nursery](https://github.com/rust-lang-nursery/) take it over?  Anyone know their criteria for inclusion?

**Comment by FredericJacobs** [comment]
@burdges: Why not have it in an org that interested contributors would manage?

**Comment by burdges** [comment]
Yes that's fine of course.  I kinda confused a github "org" with more meat space things for a moment there.  :)

**Comment by Razican** [comment]
@FredericJacobs I would say the nursery makes sense due to the relevance of the project. Having a complete crypto library could be considered as important as having a libc interface.

**Comment by burdges** [comment]
We can always move it to the nursery later once people trust the code slightly more.  I forked the repo to https://github.com/RustCryptoProject/rust-crypto and invited @FredericJacobs and @DaGenix as owners.   We can organize the existing pull requests there.  I'd hold off on doing anything with crates.io just yet.  If @DaGenix comes back in the not so distant future, then he'll find things more tidy.  If not, then one can use the repo with `rcrypto = { git = "https://github.com/RustCryptoProject/rust-crypto.git" }` or similar.

**Comment by FredericJacobs** [comment]
I'm planning on waiting until the end of the month to start working on the fork. Fingers crossed we'll hear from @DaGenix.

**Comment by DaGenix** [comment]
It is clear that I've not been able to give the project the time it needs over the last few months. I'm amenable to the idea of exploring a move under rust-lang-nursery, but, I don't know who would even be the right person to ask if that organization would be interested.

I do want to apologize for being an absent maintainer over the last few months. I'm still quite interested in helping the success of rust-crypto and discussing all options that would help with that goal.

**Comment by Razican** [comment]
Now that you are here, there is probably no need to move it to another organization, but it would be wise IMO to add at least one more maintainer to the repo, so that the project does not stall when one of the maintainers can't work on it for some time.

**Comment by burdges** [comment]
In any case, there are a few of us who could try to review and/or tweak patches so as to reduce the time @DaGenix needs to spend on them himself.

**Comment by newpavlov** [comment]
I've created [RustCrypto](https://github.com/RustCrypto) organization for my work on making rust-crypto modular. Currently only hash functions more or less ready (I recently [wrote](https://www.reddit.com/r/rust/comments/5dlui7/introducing_cryptohashes_modular_rework_of/) on reddit about it), work on other crates is ongoing.

If @DaGenix is interested he is welcome to become its another owner. Also there is open questions about how to properly organize work in this organization and about repositories structure. So if you have suggestions or can share your experience I will be happy to hear you.

P.S.: I would like to rename this organization to rust-crypto if possible, but this name was already occupied by someone, so if you are owner or know this person, please contact me.
